nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fquantum%2Fapplications%2Fnodetypeimplementations/QHAna-PluginRunner-Implementation_QAHana-ConnectTos-w2-wip1/NodeTypeImplementation.tosca

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/quantum/applications/nodetypeimplementations" id="quantumApplicationNodeTypeImplementations-QHAna-PluginRunner-Implementation_QAHana-ConnectTos-w2-wip1">
+    <NodeTypeImplementation xmlns:quantumApplicationNodeTypes="https://ust-quantil.github.io/quantum/applications/nodetypes" targetNamespace="https://ust-quantil.github.io/quantum/applications/nodetypeimplementations" name="QHAna-PluginRunner-Implementation_QAHana-ConnectTos-w2-wip1" abstract="no" final="no" nodeType="quantumApplicationNodeTypes:QHAna-PluginRunner_latest-w2-wip1">
```

Comment:
s.a.

However, release this as the last element, as the references need to be updated first.